VIX-3406 Add Marquee Effect

johncbaur opened the pull request:
VIX-3406 Add Marquee Effect

johncbaur commented:
I added a checkbox to this effect that toggle between the some of the controls being direct pixel adjustments vs a percentage of the prop.  I am wondering if I should remove the check box and just always use the percentage.

jeffu231 commented:
> I added a checkbox to this effect that toggle between the some of the controls being direct pixel adjustments vs a percentage of the prop. I am wondering if I should remove the check box and just always use the percentage.

I think it should always be based on percentage instead of pixels, so it that it scales proportionately no matter what element it is placed on.

johncbaur commented:
I removed the checkbox and now relevant fields are always a percentage of the display element.  I also fixed a bug I found while trying to see if the effect would make the same interesting patterns as the video for the other sequencer.
